plugins/package-managers/go/src/main/kotlin/GoMod.kt

```diff
@@ -215,12 +215,18 @@ class GoMod(
     }
 
     /**
-     * Return the list of all modules available via the `go list` command.
+     * Return the module information for [packages] in [projectDir].
      */
-    private fun getModuleInfos(projectDir: File): List<ModuleInfo> {
-        val list = runGo("list", "-m", "-json", "-buildvcs=false", "all", workingDir = projectDir)
+    private fun getModuleInfo(projectDir: File, vararg packages: String): Sequence<ModuleInfo> {
```

Comment:
The plural 's' refered to multiple instances of `ModuleInfo`, imo it makes sense to keep it, because it makes it clearer that multiple things are returned. What do you think? (The class does not have the `s` so we should be fine on that regard)